package.json

```diff
@@ -58,6 +58,10 @@
       "global": {
         "lines": 90
       }
+    },
+    "snapshotFormat": {
```

Comment:
The snapshot format defaults changed. Rather than recording new snapshots, which risks missing a changed snapshot, I coded in the old defaults. We can change these later, if we want.

Reply:
I suppose that now that we have run tests on it we can revert this change and update snapshots as well, right?

Reply:
Yep, we could certainly do that